QUICKSTART.md

```diff
@@ -0,0 +1,94 @@
+These are instructions on how to get a youtube instance of moderator running on a Google Cloud compute VM.
+
+Step 0:
+-------
+
+Create a Google cloud project and a VM running 'Ubuntu 18.04 LTS Minimal' in the [Google console](https://console.cloud.google.com/compute/instances).
+
+You'll also need to create a [firewall rule](https://console.cloud.google.com/networking/firewalls/list) to allow
+traffic on ports 8000 and 8080, and add that rule to your VM network tags.
+
+You'll also need to allocate a domain name for your new VM  - unfortunately the Google OAuth servers won't work with IP addresses.
+
```

Comment:
Nothing to change here (for now), but I think we could skip the domain step, and the firewall config, if we use Appengine for Node. Thoughts? I guess this provide a more generic VM/docker based way to do things.

Reply:
I'm not sure if this is true.  The problem is all client-side: client does a redirect to the Google OAuth server to bring up the OAuth login dialogue. On success, OAuth server redirects to one of the approved URLs. 

The problem is all about registering this approved URL with the OAuth server.  Ideally, we'd be able to script it, but I haven't found a mechanism for doing so yet.  (I would be very surprised if it doesn't exist though.)  Once I've discovered the mechanism, it should work for all (gcloud) deployments.  

And app-engine deployments would give us the domain name for free, so it does help a bit.

And maybe app-engine URLS are pre-approved by Google OAuth server....